tests/Predis/Connection/Replication/MasterSlaveReplicationTest.php

```diff
@@ -268,12 +268,12 @@ public function testDisconnectForcesCurrentConnectionToDisconnect(): void
     {
         $master = $this->getMockConnection('tcp://127.0.0.1:6379?role=master');
         $master
-            ->expects($this->once())
+            ->expects($this->exactly(2))
             ->method('disconnect');
 
         $slave = $this->getMockConnection('tcp://127.0.0.1:6380?role=slave');
         $slave
-            ->expects($this->once())
+            ->expects($this->exactly(2))
```

Comment:
Why is this changing?

Reply:
Same question, why `disconnect` should be called twice per connection?

Reply:
Now it's also called in `AbstractConnection` destructor

Reply:
We can't call disconnect twice.